src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Area {
```

Comment:
```suggestion
public class Circle extends Figure {
```
we should implement Area interface in class Figure